data/om.json

```diff
@@ -0,0 +1,9491 @@
+                        [
+                            "252147",
+                            "0553247"
+                        ],
+                        [
+                            "251947",
+                            "0553442"
+                        ],
+                        [
+                            "251531",
+                            "0553524"
+                        ],
+                        [
+                            "251115",
+                            "0552946"
+                        ],
+                        [
+                            "251147",
+                            "0552343"
+                        ],
+                        [
+                            "251427",
+                            "0552342"
+                        ],
+                        [
+                            "251802",
+                            "0552901"
+                        ],
+                        [
+                            "252147",
+                            "0553247"
+                        ]
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "OMDB CTA12 AMR",
+            "group": "APP",
+            "owner": [
+                "MR",
+                "DA",
+                "AE1"
+            ],
+            "sectors": [
+                {
+                    "min": 15,
+                    "max": 154,
+                    "points": [
+                        [
+                            "250040",
+                            "0553830"
+                        ],
+                        [
+                            "244730",
+                            "0555927"
+                        ],
+                        [
+                            "245559",
+                            "0555759"
+                        ],
+                        [
+                            "250541",
+                            "0554849"
+                        ],
+                        [
+                            "250813",
+                            "0554318"
+                        ],
+                        [
+                            "251055",
+                            "0553323"
+                        ],
+                        [
+                            "251115",
+                            "0552946"
+                        ],
+                        [
+                            "250600",
+                            "0552750"
+                        ],
+                        [
+                            "250040",
+                            "0553830"
+                        ]
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "OMAA CTA W2",
+            "group": "APP",
+            "owner": [
+                "ARW",
+                "ARC",
+                "AE2",
+                "AE1",
+                "GULFE",
+                "GULF"
+            ],
```

Comment:
These levels were parsed from the sector file, but I think GULF shouldn't cover this low.

Reply:
That is correct. GULF Now covers from FL245 and above and no longer has anything to do with the approaches.